### DISCUSSION TITLE
Error message: Too many levels of symbolic links

### DISCUSSION THREAD

**Discussion by TheCasualObserver** [zrepl/zrepl#578] [Q&A] [answered]
Does anybody else have experienced something like this with zrepl version=v0.5.0 or is this rather a local problem:
```
root@pc:/home/.zfs/snapshot# ls -la zrepl_20220124_002845_000
ls: cannot access 'zrepl_20220124_002845_000': Too many levels of symbolic links
```
I would like to be able to access previous snapshots.

I can access `/.zfs/snapshot/*` without problems.

**Comment by problame** [comment, marked as answer]
Whatever it is, it would be a ZFS issue, not a zrepl issue.

**Reply by TheCasualObserver** [reply]
Thanks!
It looks like I am experiencing https://github.com/openzfs/zfs/issues/11563
